crypto/getenv.c

```diff
@@ -13,9 +13,45 @@
 
 #include <stdlib.h>
 #include "internal/cryptlib.h"
+#include "e_os.h"
 
 char *ossl_safe_getenv(const char *name)
 {
+#if (defined(OPENSSL_SYS_WINDOWS)) && defined(CP_UTF8) && !defined(_WIN32_WCE)
+
+    if (GetEnvironmentVariableW(L"OPENSSL_WIN32_UTF8", NULL, 0) == 0)
+        return getenv(name);
```

Comment:
For this case I think it should probably just fall through to the bottom and run the old code..

Reply:
Yes. If user not have defined ENV OPENSSL_WIN32_UTF8, then run old code.

Reply:
I am talking about the line
```
return getenv(name);
```

Reply:
for Windows OPENSSL_issetugid always return 0, so last return getenv(name) is always execute

Reply:
ok